components/Metadata/MetadataChat.tsx

```
@@ -0,0 +1,75 @@
+import React, { useEffect, useRef } from 'react'
+import { type Message } from '@ai-sdk/react'
+import ChatMarkdown from '../ChatPage/ChatMarkdown'
```

ESLint

[error] 'ChatMarkdown' is defined but never used.

(@typescript-eslint/no-unused-vars)